Security.Platform.iOS/AgsSec.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using AeroGear.Mobile.Core.Utils;
+using AeroGear.Mobile.Security;
+
+namespace Security.Platform
+{
+    public class AgsSec
+    {
+        public AgsSec()
```

Comment:
same as my comment above

Reply:
I had your exact same feeling. I used this name because it is the same we used in iOS.
Will rename it, I think we should write down some rule about naming.